cellml1to2.xsl

```diff
@@ -52,10 +52,19 @@
 	<xsl:template match="cellml10:model | cellml11:model">
 		<model xmlns="http://www.cellml.org/cellml/2.0#" xmlns:cellml="http://www.cellml.org/cellml/2.0#">
 			<xsl:copy-of select="@name"/>
+			<!-- Units elements can only be declared as a child of the model element -->
+			<xsl:for-each select="cellml10:component/cellml10:units | cellml11:component/cellml11:units" >
+				<xsl:element name="{local-name()}" namespace="http://www.cellml.org/cellml/2.0#">
+					<xsl:apply-templates select="@* | node()"/>
```

Comment:
I _think_ this will result in attributes on the `units` elements being ignored, since the template for `@*` above does nothing. Have you checked this?

Reply:
You're correct, now I have added a copy command to copy the name attribute. Testing with the van der pol model seems to give the correct result now.